pom.xml

```diff
@@ -356,6 +297,52 @@
 				</plugins>
 			</build>
 		</profile>
+
+		<profile>
+			<id>embedded-cassandra</id>
+
+			<properties>
+				<build.cassandra.type>embedded</build.cassandra.type>
+			</properties>
+
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>build-helper-maven-plugin</artifactId>
+						<version>1.8</version>
+						<executions>
+							<execution>
+								<id>reserve-network-port</id>
+								<goals>
+									<goal>reserve-network-port</goal>
+								</goals>
+								<phase>generate-test-resources</phase>
+								<configuration>
+									<portNames>
+										<portName>build.cassandra.native_transport_port</portName>
+										<portName>build.cassandra.rpc_port</portName>
+										<portName>build.cassandra.storage_port</portName>
+										<portName>build.cassandra.ssl_storage_port</portName>
+									</portNames>
+								</configuration>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>external-cassandra</id>
+
+			<properties>
+				<build.cassandra.mode>external</build.cassandra.mode>
```

Comment:
Perhaps we should make this build property consistent with the corresponding `embedded cassandra` property `build.cassandra.type`.